lib/clickhouse/connection/client.rb

```diff
@@ -47,21 +47,22 @@ def ensure_authentication
         client.basic_auth(username || "default", password) if username || password
       end
 
-      def path(query)
+      def path(query, query_params = {})
         params = @config.select{|k, _v| k == :database}
         params[:query] = query
         params[:output_format_write_statistics] = 1
+        params.merge!(query_params)
```

Comment:
Reading the [docs](https://clickhouse.com/docs/en/interfaces/http#cli-queries-with-parameters), the params should be prefixed by `param_`, so `id` becomes `param_id`. Should we do this here to abstract the details away from the application?

Reply:
OK, lts do it here instead of client's code